### PR TITLE
Document version pinning

### PR DESCRIPTION
**Pull request by denis-yuen**
For https://ucsc-cgl.atlassian.net/browse/DOCK-1672

### CODE REVIEW

File: docs/faq.rst [hunk 1]
```diff
@@ -256,6 +256,18 @@ addresses of the users you wish to share with to give them permissions
 to your workflow. This is only available for hosted workflows and users
 with Google accounts linked to Terra.
 
+How do I add a specific version of a workflow to a collectionn and why would I want to?
```
Comment (Richard-Hansen) [inline docs/faq.rst:259]: Typo: collection*

I think this a great description of why adding workflows to a collection is valuable, but glances over the 'how' thats indicated in the title. Perhaps a step-by-step list could be included?

File: docs/faq.rst [hunk 2]
```diff
@@ -256,6 +256,18 @@ addresses of the users you wish to share with to give them permissions
...
+------------------------------------------------
+
+When creating a collection in an organization, you can pin either a specific version of a workflow or pin the workflow in general. This is a flexible system and can mean different things to different organizations. You can elaborate on what this means to you as an organization maintainer in the markdown description associated with a collection. 
+
+As an example, if your organization is responsible for a workflow and its maintenance going forward, you may want to add the collection without specifying a specific version. On the other hand, if you are creating an organization for a specific study and you are using a specific version of a workflow that has been screened for security issues and other measures of quality, you may want to pin the specific version of the workflow. You also have the option of adding a workflow to a collection multiple times by bringing up the add dialog if multiple versions of the workflow have been vetted.
```
Comment (coverbeck) [inline docs/faq.rst:264]: > you may want to add the collection without specifying a specific version

you may want to add _a workflow_ to the collection...?

File: docs/faq.rst [hunk 3]
```diff
@@ -256,6 +256,18 @@ addresses of the users you wish to share with to give them permissions
...
+
+Note that the version of a workflow can be especially important when working with launch-with partners. Some partners will take into account the version of the workflow that you are on wheras other partners will give the option of or require selecting the workflow version when performing a workflow launch. For example, Terra will automatically bring up the version you are currently browsing when performing a launch although you will have an option to override later in the process. 
+
+In summary: you can pin either a specific version of a workflow or a workflow in general depending on what relationship you wish to express, we recommend explaining further for others in the accompanying Markdown description.
```
Comment (coverbeck) [inline docs/faq.rst:268]: Feels a bit run-on sentence; I would probably make "We recommend..." a new sentence.

File: docs/faq.rst [hunk 4]
```diff
@@ -256,6 +256,18 @@ addresses of the users you wish to share with to give them permissions
 to your workflow. This is only available for hosted workflows and users
 with Google accounts linked to Terra.
 
+How do I add a specific version of a workflow to a collectionn and why would I want to?
+------------------------------------------------
+
+When creating a collection in an organization, you can pin either a specific version of a workflow or pin the workflow in general. This is a flexible system and can mean different things to different organizations. You can elaborate on what this means to you as an organization maintainer in the markdown description associated with a collection.
```
Comment (coverbeck) [inline docs/faq.rst:262]: I don't feel too strongly about this, but elsewhere in the doc we "add" a workflow rather than "pin" it.

**Review by coverbeck — APPROVED** [review]

**Comment by NatalieEO** [comment]
Should update the instructions/pictures here https://docs.dockstore.org/en/develop/advanced-topics/organizations-and-collections.html. 
And adding to Richard's comment, but maybe the FAQ entry should just be "why" and you link out to question from the above?

File: docs/faq.rst (hunk 1 above)
Comment (denis-yuen) [reply docs/faq.rst:259]: Changed title and changed picture on the page @NatalieEO referenced

**Comment by denis-yuen** [comment]
```
Why would I want to add a specific version of a workflow to a collection?
------------------------------------------------
/root/project/docs/faq.rst:260: WARNING: Title underline too short.
```
really

**Review by Richard-Hansen — APPROVED** [review]

**Review by garyluu — APPROVED** [review]

**Review by GFJHogue — APPROVED** [review]

**Review by NatalieEO — APPROVED** [review]

**Review by coverbeck — APPROVED** [review]
Wow, did the link checker catch all those external failing links? I don't think it used to do that.

**Comment by denis-yuen** [comment]
> Wow, did the link checker catch all those external failing links? I don't think it used to do that.

Yeah, but I have no idea why me adding one link started things failing

**Pull request merged**